packages/design-system/spinner/index.web.tsx

```diff
@@ -1,11 +1,10 @@
-import { useEffect, useRef, useCallback, useMemo } from "react";
+import { useEffect, useCallback, useMemo } from "react";
 import { Animated, Easing } from "react-native";
 
 import { getSpinnerSize, SpinnerView, SpinnerProps } from "./spinner-view";
 
 export const Spinner = ({ size, duration = 750, ...rest }: SpinnerProps) => {
-  const transition = useRef(new Animated.Value(0)).current;
-
+  const transition = useMemo(() => new Animated.Value(0), []);
```

Comment:
It's better to use `useRef` for Animated.Values. It's safe to revert this change. 
Also, what do you think about using `moti` instead? 

```javascript
<MotiView 
  from={{rotate: '0deg'}}
  animate={{rotate: '360deg'}}
  transition={{
    type: 'timing',
    loop: true,
    repeatReverse: false,
    easing: Easing.linear
  }}
  style={{
    height: getSpinnerSize(size),
    width: getSpinnerSize(size),
  }}
/>
```

Reply:
thank you for your review!
it's just a test, I'm not sure if it's causing every update [this](https://www.chromatic.com/build?appId=619f8881c3b4c2003a08edf7&number=2092), `moti` I'll try again, I can't remember what problem I had before without using it.

Reply:
just try it:
![image](https://user-images.githubusercontent.com/37520667/176128666-404cf751-1f69-47ee-b925-97cb8163343b.png)
it's working on the web but does not work on native. :(

Reply:
but thank you tips!
so let's try using `moti` on the web, native code is temporarily unchanged, see [this](https://www.chromatic.com/build?appId=619f8881c3b4c2003a08edf7&number=2092) if it disappears